string group_name
string[] led_names
---
